taskcluster/ci/signing/kind.yml

```diff
@@ -36,10 +36,6 @@ job-template:
                     by-level:
                         '3': fennec-nightly-signing
                         default: dep-signing
-                nightly-simulation:
```

Comment:
nightly-sim is gonna be signed with the dummy key, aka the `dep` key so we're good to go!